src/common/chat-participants/powerpages/PowerPagesChatParticipantConstants.ts

```diff
@@ -13,3 +13,9 @@ export const AUTHENTICATION_FAILED_MSG = vscode.l10n.t('Authentication failed. P
 export const COPILOT_NOT_AVAILABLE_MSG = vscode.l10n.t('Copilot is not available. Please contact your administrator.');
 export const PAC_AUTH_NOT_FOUND = vscode.l10n.t('Active auth profile is not found or has expired. Please try again.');
 export const SUPPORTED_ENTITIES = [ADX_ENTITYFORM, ADX_ENTITYLIST];
+// Telemetry Event Names
+export const GitHubPowerPagesAgentInvoked = 'GitHubPowerPagesAgentInvoked';
```

Comment:
for events name lets follow the pattern of GITHUB_POWER_PAGES_AGENT_INVOKED - save as we do for other telemetry constants

Reply:
in fact use this qualifier: VSCODE_EXTENSION_POWER_PAGES_AGENT_INVOKED or VSCODE_EXTENSION_GITHUB_POWER_PAGES_AGENT_INVOKED